release-schema.json

```
@@ -1,5 +1,14 @@
 {
   "definitions": {
+    "Tender": {
+      "properties": {
+        "subcontractingTerms": {
+          "title": "Subcontracting terms",
+          "description": "The obligations of the suppliers who subcontract.",
```

To follow the same pattern as the contractTerms and submissionTerms extensions (make similar change to `SubcontractingTerms` definition):

```suggestion
          "description": "Information about the terms governing subcontracting.",
```